tests/tracking/test_tracking.py

```diff
@@ -431,6 +431,18 @@ def get_store_mock():
     with mock.patch("mlflow.store.file_store.FileStore.log_batch") as _get_store_mock:
         yield _get_store_mock
 
+def test_set_experiment_tags():
```

Comment:
Nit: Can we move this test case to https://github.com/mlflow/mlflow/blob/master/tests/tracking/fluent/test_fluent.py?

Can we also introduce a separate test case for `test_set_experiment_tag`?

Reply:
Moved `test_set_experiment_tags` to `test_fluent.py`, added `test_set_experiment_tag` and added both to `__all__` in `mlflow.__init__.py`